.github/workflows/labeler.yml

```diff
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
+on:
+  - pull_request
+
+permissions: write-all
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: srvaroa/labeler@v0.8
```

Comment:
Could you describe why you picked this specific action here? Github builds their own (`actions/labeler`). It would be helpful if you explained the pros and cons of that versus this one in making your decision.